docs/usage/configuration/config_documentation.md

```diff
@@ -3170,6 +3170,10 @@ Options for each entry include:
 * `client_secret`: oauth2 client secret to use. May be omitted if
   `client_secret_jwt_key` is given, or if `client_auth_method` is 'none'.
```

Comment:
We should probably mention something here too about `client_secret_path`?

Reply:
@clokep Good point, will update this as soon as we're aligned on your two notes below and fix up everything then in one go then :)